docs/src/main/asciidoc/jreleaser.adoc

```diff
@@ -107,20 +107,20 @@ shell completion scripts, man pages, license, readme, and more.
 == Creating the distribution
 
 We can leverage the link:http://maven.apache.org/plugins/maven-assembly-plugin/[maven-assembly-plugin] to create such
-a distribution. We'll also make use of the link:https://github.com/trustin/os-maven-plugin[os-maven-plugin] to properly
+a distribution. We'll also make use of the link:https://github.com/tikonsun/os-detector-maven-plugin[os-detector-maven-plugin] to properly
```

Comment:
I get a 404 when hitting https://github.com/tikonsun/os-detector-maven-plugin

Reply:
https://github.com/tisonkun/os-detector

`tisonkun` and not `tikonsun`